<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanConfig</class>
 <widget class="QDialog" name="ScanConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>291</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Scan Configuration</string>
  </property>
  <widget class="QLabel" name="UNIT_US_2">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>110</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>μs</string>
   </property>
  </widget>
  <widget class="QLabel" name="FREQ_UNIT">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>210</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>kHz</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="END_DELAY_EDIT_N">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>105</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_US_1">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>70</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>μs</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="END_DELAY_EDIT_U">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>105</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="START_DELAY_EDIT_N">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>65</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="START_DELAY_LBL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>80</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Starting</string>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_NS_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>ns</string>
   </property>
  </widget>
  <widget class="QLabel" name="STEP_DELAY_LBL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>80</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Stepping</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="REP_FREQ_EDIT">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>205</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_NS_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>ns</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="START_DELAY_EDIT_U">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>65</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="FRAME_COUNT_EDIT">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>245</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="STEP_SIZE_DELAY_EDIT">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>145</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="FREQ_LBL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>86</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Rep. freq.</string>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_1">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>150</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>ns</string>
   </property>
  </widget>
  <widget class="QLabel" name="END_DELAY_LBL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>80</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Ending</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CAPTURE_SCAN_ORI_CHK">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>original</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CAPTURE_SCAN_RES_CHK">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>result</string>
   </property>
  </widget>
  <widget class="QLabel" name="FRAME_LBL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>86</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Frames count</string>
   </property>
  </widget>
  <widget class="QLabel" name="DELAY_LBL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DELAY</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="GATE_WIDTH_LBL">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>GATE WIDTH</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="END_GW_LBL">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>80</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Ending</string>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_US_3">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>70</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>μs</string>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_NS_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>ns</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="END_GW_EDIT_U">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>105</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_2">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>150</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>ns</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="END_GW_EDIT_N">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>105</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="STEP_SIZE_GW_EDIT">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>145</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="START_GW_EDIT_U">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>65</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="STEP_GW_LBL">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>80</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Stepping</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="START_GW_EDIT_N">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>65</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_US_4">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>110</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>μs</string>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_NS_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>ns</string>
   </property>
  </widget>
  <widget class="QLabel" name="START_GW_LBL">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>80</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Starting</string>
   </property>
  </widget>
  <widget class="QLabel" name="CAPTURE_LBL">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CAPTURE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="RECORD_LBL">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RECORD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="RECORD_SCAN_RES_CHK">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>result</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="RECORD_SCAN_ORI_CHK">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>original</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="START_DELAY_EDIT_M">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>65</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="END_DELAY_EDIT_M">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>105</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_M_1">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>70</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_M_2">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>110</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="END_GW_EDIT_M">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>105</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="START_GW_EDIT_M">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>65</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_M_3">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>70</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QLabel" name="UNIT_M_4">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>110</y>
     <width>21</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QLabel" name="PTZ_LBL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PTZ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="START_PTZ_LBL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>80</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Starting</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="START_ANGLE_H_EDT">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>65</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="START_ANGLE_V_EDT">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>65</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="START_ANGLE_H">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>11</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QLabel" name="START_ANGLE_V">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>11</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="END_ANGLE_H_EDT">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>105</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="END_PTZ_LBL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>80</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Ending</string>
   </property>
  </widget>
  <widget class="QLabel" name="END_ANGLE_V">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>11</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QLabel" name="END_ANGLE_H">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>11</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="END_ANGLE_V_EDT">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>105</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="STEP_GW_LBL_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>80</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Stepping</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="STEP_ANGLE_V_EDT">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>145</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="STEP_ANGLE_H">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>11</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="STEP_ANGLE_H_EDT">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>145</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="STEP_ANGLE_V">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>11</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QLabel" name="POS_CNT_PTZ_LBL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>86</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos. count</string>
   </property>
  </widget>
  <widget class="QLabel" name="FRAME_V">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>11</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QLabel" name="FRAME_H">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>11</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FRAME_V_EDT">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>185</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="FRAME_H_EDT">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>185</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_PTZ_DIRECTION_N">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_PTZ_DIRECTION_Z">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NUM_SINGLE_POS_EDT">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="PTZ_WAIT_TIME_EDT">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PTZ_WAIT_TIME_LBL">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>225</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PTZ wait</string>
   </property>
  </widget>
  <widget class="QLabel" name="NUM_SINGLE_POS_LBL">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>225</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>num.</string>
   </property>
  </widget>
  <zorder>START_GW_EDIT_M</zorder>
  <zorder>UNIT_M_3</zorder>
  <zorder>UNIT_M_4</zorder>
  <zorder>END_GW_EDIT_M</zorder>
  <zorder>UNIT_M_2</zorder>
  <zorder>END_DELAY_EDIT_M</zorder>
  <zorder>START_DELAY_EDIT_M</zorder>
  <zorder>UNIT_M_1</zorder>
  <zorder>UNIT_US_2</zorder>
  <zorder>FREQ_UNIT</zorder>
  <zorder>END_DELAY_EDIT_N</zorder>
  <zorder>UNIT_US_1</zorder>
  <zorder>END_DELAY_EDIT_U</zorder>
  <zorder>START_DELAY_EDIT_N</zorder>
  <zorder>START_DELAY_LBL</zorder>
  <zorder>UNIT_NS_1</zorder>
  <zorder>STEP_DELAY_LBL</zorder>
  <zorder>REP_FREQ_EDIT</zorder>
  <zorder>UNIT_NS_2</zorder>
  <zorder>START_DELAY_EDIT_U</zorder>
  <zorder>FRAME_COUNT_EDIT</zorder>
  <zorder>STEP_SIZE_DELAY_EDIT</zorder>
  <zorder>FREQ_LBL</zorder>
  <zorder>UNIT_1</zorder>
  <zorder>END_DELAY_LBL</zorder>
  <zorder>CAPTURE_SCAN_ORI_CHK</zorder>
  <zorder>CAPTURE_SCAN_RES_CHK</zorder>
  <zorder>FRAME_LBL</zorder>
  <zorder>DELAY_LBL</zorder>
  <zorder>GATE_WIDTH_LBL</zorder>
  <zorder>END_GW_LBL</zorder>
  <zorder>UNIT_US_3</zorder>
  <zorder>UNIT_NS_4</zorder>
  <zorder>END_GW_EDIT_U</zorder>
  <zorder>UNIT_2</zorder>
  <zorder>END_GW_EDIT_N</zorder>
  <zorder>STEP_SIZE_GW_EDIT</zorder>
  <zorder>START_GW_EDIT_U</zorder>
  <zorder>STEP_GW_LBL</zorder>
  <zorder>START_GW_EDIT_N</zorder>
  <zorder>UNIT_US_4</zorder>
  <zorder>UNIT_NS_3</zorder>
  <zorder>START_GW_LBL</zorder>
  <zorder>CAPTURE_LBL</zorder>
  <zorder>RECORD_LBL</zorder>
  <zorder>RECORD_SCAN_RES_CHK</zorder>
  <zorder>RECORD_SCAN_ORI_CHK</zorder>
  <zorder>PTZ_LBL</zorder>
  <zorder>START_PTZ_LBL</zorder>
  <zorder>START_ANGLE_H_EDT</zorder>
  <zorder>START_ANGLE_V_EDT</zorder>
  <zorder>START_ANGLE_H</zorder>
  <zorder>START_ANGLE_V</zorder>
  <zorder>END_ANGLE_H_EDT</zorder>
  <zorder>END_PTZ_LBL</zorder>
  <zorder>END_ANGLE_V</zorder>
  <zorder>END_ANGLE_H</zorder>
  <zorder>END_ANGLE_V_EDT</zorder>
  <zorder>STEP_GW_LBL_2</zorder>
  <zorder>STEP_ANGLE_V_EDT</zorder>
  <zorder>STEP_ANGLE_H</zorder>
  <zorder>STEP_ANGLE_H_EDT</zorder>
  <zorder>STEP_ANGLE_V</zorder>
  <zorder>POS_CNT_PTZ_LBL</zorder>
  <zorder>FRAME_V</zorder>
  <zorder>FRAME_H</zorder>
  <zorder>FRAME_V_EDT</zorder>
  <zorder>FRAME_H_EDT</zorder>
  <zorder>RADIO_PTZ_DIRECTION_N</zorder>
  <zorder>RADIO_PTZ_DIRECTION_Z</zorder>
  <zorder>NUM_SINGLE_POS_EDT</zorder>
  <zorder>PTZ_WAIT_TIME_EDT</zorder>
  <zorder>PTZ_WAIT_TIME_LBL</zorder>
  <zorder>NUM_SINGLE_POS_LBL</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
